react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+import CustomButton from './CustomButton';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer">
+          <div>
+            <CustomButton text={'i'} color="purple" />
```

Comment:
If you are just passing a string, you should pass it using the normal syntax (like you did the colors):
`<CustomButton text="i" color="purple" />`
You only need the `{}` when you are passing a variable or expression.